jenkins-slaves/jenkins-slave-helm/Dockerfile

```diff
@@ -12,4 +21,40 @@ RUN curl -skL -o /tmp/helm.tar.gz https://get.helm.sh/helm-v${VERSION}-linux-amd
     curl -sL  https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_amd64 -o /usr/local/bin/yq && \
     chmod -R 775 /usr/local/bin/yq
 
+## Install ct
+RUN curl -sL -o /tmp/chart-testing.tar.gz https://github.com/helm/chart-testing/releases/download/v${CT_VERSION}/chart-testing_${CT_VERSION}_linux_amd64.tar.gz && \
+    mkdir /tmp/chart-testing && \
+    tar -C /tmp/chart-testing -zxf /tmp/chart-testing.tar.gz && \
+    mv /tmp/chart-testing/ct /usr/local/bin && \
+    chmod 775 /usr/local/bin/ct && \
+    rm /tmp/chart-testing.tar.gz && \
+    mkdir ${HOME}/.ct && \
+    mv /tmp/chart-testing/etc/chart_schema.yaml /tmp/chart-testing/etc/lintconf.yaml ${HOME}/.ct/ && \
+    rm -rf /tmp/chart-testing
+
+## Install python 3.6, yamale, and yamllint
+RUN INSTALL_PKGS="rh-python36 rh-python36-python-pip" && \
+    yum -y --setopt=tsflags=nodocs install $INSTALL_PKGS && \
+    yum -y clean all --enablerepo='*' && \
+    source scl_source enable rh-python36 && \
+    python3 -m pip install yamale==3.0.1 && \
+    python3 -m pip install yamllint==1.24.1
+
+## Persistently enable software collections
+RUN mkdir /usr/share/container-scripts
+COPY scl_enable /usr/share/container-scripts
+RUN chmod 555 /usr/share/container-scripts/scl_enable
+ENV BASH_ENV=/usr/share/container-scripts/scl_enable \
+    ENV=/usr/share/container-scripts/scl_enable \
+    PROMPT_COMMAND=". /usr/share/container-scripts/scl_enable"
+
+## Install git > 2.17 (ubi repos don't provide this)
```

Comment:
any reason for git 2.24? since you are using scl, would work?
- https://www.softwarecollections.org/en/scls/rhscl/rh-git218/

Reply:
I tried to use rh-git218, but I don't think the UBI repos have it. I also browsed through them, but didn't see any rpms for git.

ct has a dependency on git 2.17 or greater (https://github.com/helm/chart-testing#prerequisites)

Reply:
oh, I get what you're saying. I might be able to try this here - https://www.softwarecollections.org/en/docs/

Reply:
Updated to use rh-git218